autonomy/cli/helpers/docstring.py

```diff
@@ -57,7 +57,7 @@ def analyse_docstrings(
     :param update: whether to update the content if required
     :param packages_dir: Path to packages directory
 
-    :return: boolean specifyning whether the update is needed or not
+    :return: boolean specifying whether the update is needed or not
```

Comment:
all hashes changed because I fixed a type. Last time in my life i will do that

Reply:
going to push on top a new commit putting back the typo because this is just too terrible.